licensing/locale/fr_FR/LC_MESSAGES/messages.po

```diff
@@ -712,12 +712,12 @@ msgstr "Les utilisateurs 'Restricted' ne peuvent pas consulter les documents"
 msgid ""
 "View only users can view all license information, including the license pdf"
 msgstr ""
-"Les utilisateurs 'View only' peuvent consulter tous les détails de licences, "
-"y compris le fichier PDF de la licence"
+"Les utilisateurs 'View only' peuvent consulter tous les détails des "
+"licences, y compris le fichier PDF de la licence"
 
 #: ajax_forms.php:951
 msgid "Terms Tool Email"
-msgstr "Envoyer un email Outil des termes"
+msgstr "Adresse email de l'Outil des termes"
```

Comment:
The great majority of all the references to the Terms Tool and the other modules are capitalized in english also. So that would be a separate issue about how we name the modules in the GUI.
Should I create it?